packages/app/src/pages/Specs/Runner.vue

```diff
@@ -51,6 +51,18 @@ const query = useQuery({
   pause: isRunMode && window.top === window,
 })
 
+const queryInterval = setInterval(() => {
```

Comment:
Show this be conditional based on `isRunMode`? We don't want to have an interval doing anything graphql related since runMode doesn't need graphql.

Reply:
good point, will add that check